Eatery Blue/UI/SettingsMain/SettingsMainMenuModelController.swift

```diff
@@ -52,18 +52,14 @@ class SettingsMainMenuModelController: SettingsMainMenuViewController {
             }
         ))
         setCustomSpacing(24)
-        if let credentials = try? NetIDKeychainManager.shared.get() {
-            var didAttemptLogOut = false
 
-            addLoginStatusView(credentials.netId, logOut: {
-                if !didAttemptLogOut {
-                    Task {
-                        await Networking.default.logOut()
-                    }
-                    didAttemptLogOut = true
-                }
-            })
-        }
-    }
+        var didAttemptLogOut = false
 
+        addLoginStatusView(logOut: {
+            if !didAttemptLogOut {
+                Networking.default.logOut()
```

Comment:
If we call Networking's shared instance 'default' maybe we should also change Keychain Access's shared instance to 'default' instead of 'shared' too

Reply:
ideally we'll change networking's shared instance as the next task, it's not done very well right now :(